.circleci/config.yml

```diff
@@ -280,7 +280,12 @@ step-maybe-electron-dist-strip: &step-maybe-electron-dist-strip
     command: |
       if [ "$STRIP_BINARIES" == "true" ] && [ "`uname`" != "Darwin" ]; then
         cd src
-        electron/script/strip-binaries.py --target-cpu="$TARGET_ARCH"
+        mkdir out/Default/debug
+        # Generate electron.debug. Required to do it from there in order to maintain debug references.
+        cd out/Default/debug
```

Comment:
Generally we prefer to make our scripts not need to be run in a particular directory, and ideally they'll create their own directories if they don't exist (which makes it easier to run them manually). Can you make the scripts take an argument for the debug out directory, & make them agnostic as to where they're run from?

e.g.

```
electron/script/copy-debug-symbols.py --target-cpu="$TARGET_ARCH" --compress --out-dir out/Default/debug
```

Reply:
I think I can make `copy-debug-symbols.py` agnostic as to where it is run from but not for add-debug-link. Last time I tried the result was not usable so my assumption is that it was required to run it where the debug file live. But I will give another try. Thx for your comments!

Reply:
the script itself can definitely be made agnostic, if necessary you can add the relevant working directory arguments to the python `subprocess` invocations.

Reply:
Done.